packages/mural-canvas/src/components/canvas/index.tsx

```diff
@@ -72,6 +76,24 @@ export class CanvasHost extends React.Component<PropTypes> {
     }
   };
 
+  handleRpcMessage = async (message: RpcMessage) => {
+    const { apiClient, rpcClient } = this.props;
+
+    if (!rpcClient) return;
```

Comment:
**error**: I think we should `throw new Error('Broken RPC connection, rpcClient missing.');

This is the kind of "WHAT THE HELL" moment that will break behavior anyways, so might as well surface the issue.